features/users/authenticate_users.feature

```diff
@@ -130,6 +201,40 @@ Feature: User Authentication
       And I should not see "Log Out"
       And I should not see "Preferences"
 
+  Scenario: Login case (in)sensitivity
+    Given the following activated user exists
+      | login      | password |
+      | TheMadUser | password1 |
+    When I am on the home page
+      And I fill in "User name" with "themaduser"
+      And I fill in "Password" with "password1"
+      And I press "Log In"
+    Then I should see "Successfully logged in."
+      And I should see "Hi, TheMadUser!"
+
+  Scenario: Login with email or login
+    Given the following activated user exists
+      | login      | email                          | password |
+      | TheMadUser | themaduser@archiveofourown.org | password |
```

Comment:
We should use `@example.com` email addresses when adding new tests (although this one's not too bad since we own the domain and know no one is using it).